libaleth-interpreter/VM.cpp

```diff
@@ -806,9 +807,9 @@ void VM::interpretCases()
             if (m_SP[0] < 31)
             {
                 unsigned testBit = static_cast<unsigned>(m_SP[0]) * 8 + 7;
-                u256& number = m_SP[1];
-                u256 mask = ((u256(1) << testBit) - 1);
-                if (boost::multiprecision::bit_test(number, testBit))
+                intx::uint256& number = m_SP[1];
+                intx::uint256 mask = ((intx::uint256(1) << testBit) - 1);
```

Comment:
`1_u256`? But I think it requires `using namespace intx;`, so not sure if it's worth it.
But `using namespace intx;` could be here in the local scope